frontend/server/src/Controllers/User.php

```diff
@@ -3946,6 +3947,48 @@ public static function getIndexDetailsForTypeScript(\OmegaUp\Request $r) {
             \OmegaUp\Controllers\User::ALLOWED_CODER_OF_THE_MONTH_CATEGORIES
         );
 
+        $token = $r->ensureOptionalString(
+            'token',
```

Comment:
```suggestion
            'parental_verification_token',
```

"token" can mean a lot of things, so let's try to be explicit about its purpose.